Clarifying language re: spec plans vs applies (#1749)
hashicorp/terraform-website: content/source/docs/cloud/run/ui.html.md

    -> **Note:** If Terraform Cloud skips a plan because the changes weren't relevant, it will still post a passing commit status to the pull request.

> - Status checks on the pull request are not updated after an apply, so a commit with a successful plan but an errored apply will still show the passing commit status relevant to the plan.

Reviewer: Hmm, took me a few passes to understand that, but it does seem like an important thing to point out. How about this? 

```suggestion
- Commit statuses only show the results of speculative plans. If a pull request is merged and results in a normal (non-speculative) run, Terraform Cloud will not update the pull request's status checks with the results (for example, if an apply partially fails).
```

Or that might prompt you to think of a better way to describe that; either way, I'll go ahead and pre-approve.

> - Status checks on the pull request are not updated after an apply, so a commit with a successful plan but an errored apply will still show the passing commit status relevant to the plan.

Reviewer 2: Just a suggestion to remove some passive voice, and a clarification. Is the apply we're talking about here the apply that results from merging the PR itself?

```suggestion
- Terraform Cloud does not update the status checks on the pull request with the status of the associated apply. A commit with a successful plan but an errored apply will still show the passing commit status from the plan.
```

Author: > Just a suggestion to remove some passive voice, and a clarification. Is the apply we're talking about here the apply that results from merging the PR itself?

Yes! That's correct. I'm going to propose another version of this that merges both you and Nick's comments:

```suggestion
- Terraform Cloud does not update the status checks on a pull request with the status of an associated apply. This means that a commit with a successful plan but an errored apply will still show the passing commit status from the plan.
```

Sound good @nfagerlund @judithpatudith ?
